README.md

```diff
@@ -17,21 +17,21 @@ Collection of Maven plug-ins and archetypes for managing WebSphere Application S
 
 ## Build
 
-Use Maven 3.x to build the Liberty plug-ins and archetypes.
+Use Maven 3.5.0 or later to build the Liberty plugins and archetypes.
 
-* `mvn install` : builds the plug-in and the archetypes.
-* `mvn install -Poffline-its -DlibertyInstallDir=<liberty_install_directory>` : builds the plug-in and the archetypes and runs the integration tests by providing an existing installation.
-* `mvn install -Ponline-its -Druntime=<ol|wlp> -DruntimeVersion=<runtime_version>` : builds the plug-in and archetypes and runs the integration tests by downloading a new server. Set runtime to `ol` to run tests using the Open Liberty runtime, or `wlp` to run tests using the WLP Java EE 7 runtime.
+* `mvn install` : builds the plugin and the archetypes.
+* `mvn install -Poffline-its -DlibertyInstallDir=<liberty_install_directory>` : builds the plugin and the archetypes and runs the integration tests by providing an existing installation.
+* `mvn install -Ponline-its -Druntime=<ol|wlp> -DruntimeVersion=<runtime_version>` : builds the plugin and archetypes and runs the integration tests by downloading a new server. Set runtime to `ol` to run tests using the Open Liberty runtime, or `wlp` to run tests using the WebSphere Liberty Java EE 7 runtime.
 
-## Plug-ins
+## plugins
 
-### liberty-maven-plugin
+### Liberty Maven Plugin
```

Comment:
This is referenced in the table of contents up front. Should you change that reference to say "Liberty Maven Plugin" also?

Reply:
Done.